index.html

```diff
@@ -7,26 +7,47 @@
 		<title>Awesome Books</title>
 	</head>
 	<body>
-		<h1>Awesome Books</h1>
-		<section id="books"></section>
+		<section id="header">
+			<h2>Awesome Books</h2>
+			<ul class="ul">
+				<li id="list" class="li">List of Awesome books</li>
+				<li id="addnew" class="li">Add new Book</li>
+				<li id="contact" class="li">Contact info</li>
+			</ul>
+		</section>
+		<section id="books">
+			<h1>All Awesome Books</h1>
+
+		</section>
```

Comment:
### Optional
- [ ] It would be better to display only one page at a time when the page is refreshed. Although on page load only one of the component are displayed at a time
![Uploading Screenshot from 2023-07-07 23-06-49.png…]()